core/html_api.php

```diff
@@ -311,6 +311,22 @@ function html_meta_redirect( $p_url, $p_time = null, $p_sanitize = true ) {
 	return true;
 }
 
+/**
+ * Print a canonical meta tag.
+ *
+ * @param string  $p_url      The canonical URL: has to be a relative path.
+ * @return boolean
```

Comment:
Why set a return type ? It is not being used.

Reply:
Was just copying `html_meta_redirect`, but in hindsight that had a reason.